src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,29 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:gs="http://namespaces.zope.org/genericsetup">
+
+  <include file="profiles.zcml" />
+
+  <gs:registerProfile
```

Comment:
@hvelarde This is indeed nice and have been used by Plone in plone.app.upgrades. https://github.com/plone/plone.app.upgrade/blob/c758c0f61f1112bc283c0c05f4e9da69c05f4d48/plone/app/upgrade/v43/profiles.zcml